notebooks/en/annotate_text_data_transformers_via_active_learning.ipynb

```diff
@@ -0,0 +1,2296 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    pd.set_option('max_colwidth', None)
</pre><p>nit would you be able to move this to the bottom of the imports cell? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/63/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Done!</p>